src/Kafka/Consumer/ConsumerProperties.hs

```diff
@@ -36,11 +38,12 @@ data ConsumerProperties = ConsumerProperties
   { cpProps     :: Map Text Text
   , cpLogLevel  :: Maybe KafkaLogLevel
   , cpCallbacks :: [KafkaConf -> IO ()]
+  , cpUserPolls :: Any
```

Comment:
Using `Any` means that once stitched on it can never be switched off. It could be confusing, so can we have just `Bool` or `Last Bool`?